packages/@react-aria/table/src/useTable.ts

```diff
@@ -15,18 +15,21 @@ import {GridAria, GridProps, useGrid} from '@react-aria/grid';
 import {gridIds} from './utils';
 // @ts-ignore
 import intlMessages from '../intl/*.json';
+import {Key, RefObject, useMemo} from 'react';
 import {Layout} from '@react-stately/virtualizer';
-import {mergeProps, useDescription, useId, useUpdateEffect} from '@react-aria/utils';
+import {mergeDescriptions, mergeProps, useDescription, useId, useUpdateEffect} from '@react-aria/utils';
 import {Node} from '@react-types/shared';
-import {RefObject, useMemo} from 'react';
 import {TableKeyboardDelegate} from './TableKeyboardDelegate';
 import {TableState} from '@react-stately/table';
 import {useCollator, useLocale} from '@react-aria/i18n';
+import {useInteractionModality} from '@react-aria/interactions';
 import {useMessageFormatter} from '@react-aria/i18n';
 
 interface TableProps<T> extends GridProps {
   /** The layout object for the table. Computes what content is visible and how to position and style them. */
-  layout?: Layout<Node<T>>
+  layout?: Layout<Node<T>>,
+  /** Handler that is called when a user performs an action on a row. */
+  onAction?: (key: Key) => void
```

Comment:
Hmmmmmmm. So this prop is used in Spectrum, but in Aria we documented it as passing the `onAction` directly to `useGridRow` and `useGridCell`. That way you can control which rows it's on and whether it goes on rows or cells. I guess we might need to find a way for useTable to know that a description is needed though...